src/ViewHelpers/ConstantViewHelper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+/**
+ * Wrapper for PHPs :php:`constant` function.
+ * See https://www.php.net/manual/function.constant.php.
+ *
+ * Examples
+ * ========
+ *
+ * Get built-in PHP constant
+ * -------------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: 'PHP_INT_MAX')}
+ *
+ * Output::
+ *
+ *    9223372036854775807
+ *    (Depending on CPU architecture).
+ *
+ * Get class constant
+ * ------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Class::CONSTANT')}
+ *
+ * Get enum value
+ * --------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Enum::CASE')}
```

Comment:
In my opinion, this leads to the misuse of enums. 
If these are used in a condition, for example, the enums should be compared and not the "backed value", which would be the case here. The enum is then cast into a "string" here. This works, but in my opinion it should not be used like this.

Reply:
Ah I was wrong. Passing a enum to "constant()" does return the ENUM and not the "backed value". So ok, then I'm fine with it even if i would not expect that in the first place from a "Constant" ViewHelper.

So in this case it woult also be possible to call methods on the enum (if they're using a method name callable by fluid like "isXXX" or "getXXX")

Reply:
I think that this is not a big problem in the current implementation. I would leave it as-is.